samples/cellular/modem_shell/src/ping/icmp_ping.c

```diff
@@ -503,8 +514,7 @@ static uint32_t send_ping_wait_reply(struct icmp_ping_shell_cmd_argv *ping_args,
 
 static void icmp_ping_tasks_execute(struct icmp_ping_shell_cmd_argv *ping_args)
 {
-	struct addrinfo *si = ping_args->src;
-	struct addrinfo *di = ping_args->dest;
+	struct icmp_ping_shell_cmd_argv used_ping_args;
```

Comment:
Would `current_ping_args` be a more descriptive name?

Reply:
yes, changed.